docs/4-enable-audio.md

```diff
@@ -0,0 +1,144 @@
+        this.audio.src = window.URL.createObjectURL(stream);
+      }
+
+      this.audio.onloadedmetadata = () => {
+        this.audio.play();
+      }
+
+      this.eventLogger('member:media')()
+    }).catch(this.errorLogger)
+  })
+}
+```
+
+### 1.3 - Add disable audio handler
+
+Next, we'll add the ability for a user to disable the audio stream as well. In order to do this, we'll update the `setupUserEvents` method to trigger `conversation.audio.disable()` when the user clicks the `Disable Audio` button.
+
+```javascript
+setupUserEvents() {
+...
+  this.disableButton.addEventListener('click', () => {
+    this.conversation.audio.disable().then(this.eventLogger('member:media')).catch(this.errorLogger)
+  })
+}
+```
+
+### 1.4 - Add member:media listener
+
+With these first parts we're sending `member:media` events into the conversation. Now we're going to register a listener for them as well that updates the `messageFeed`. In order to do that, we'll add a listener for `member:media` events at the end of the `setupConversationEvents` method
+
+```javascript
+setupConversationEvents(conversation) {
+  ...
+
+  conversation.on("member:media", (from, media) => {
+    console.log(`*** Member changed media state`, from, media)
+    const text = `${from.name} <b>${media.audio ? 'enabled' : 'disabled'} audio in the conversation</b><br>`
+    this.messageFeed.innerHTML = text + this.messageFeed.innerHTML
+  })
+
+}
+```
+
+If we want the conversation history to be updated, we need to add a case for `member:media` in the `showConversationHistory` switch:
+
+```javascript
+showConversationHistory(conversation) {
+  ...
+  switch (events[Object.keys(events)[i - 1]].type) {
+    ...
+    case 'member:media':
+      eventsHistory += `${conversation.members[events[Object.keys(events)[i - 1]].from].name} @ ${date}: <b>${events[Object.keys(events)[i - 1]].body.audio ? "enabled" : "disabled"} audio</b><br>`
+      break;
+    ...
+  }
+}
+```
+
+
+### 1.5 - Open the conversation in two browser windows
+
+Now run `index.html` in two side-by-side browser windows, making sure to login with the user name `jamie` in one and with `alice` in the other. Enable audio on both and start talking. You'll also see events being logged in the browser console.
+
+Thats's it! Your page should now look something like [this](../examples/4-enable-audio/index.html).
+
+## Where next?
+
+- Have a look at the [Nexmo Conversation JS SDK API Reference](https://ea.developer.nexmo.com/sdk/conversation/javascript/)
+
+## ICE Servers
+If for some reason the WebRTC P2P connection doesn't work on your network, you need to overwrite the [ICE Servers](https://en.wikipedia.org/wiki/Interactive_Connectivity_Establishment) when you create you `ConversationClient` instance:
+
+```javascript
+new ConversationClient({
+  debug: false,
+  iceServers: {
+    urls: 'turn:138.68.169.35:3478?transport=tcp',
+    credential: 'bar',
+    username: 'foo2'
+  }
+})
+```
```

Comment:
I think it'd be awesome to have a link to a video here showing it off

Reply:
I think that's a great idea. I'll record when I get back in the office